### PR TITLE
Aggregate Test Results

### PR DESCRIPTION
**Pull request by trieloff**
Wouldn't it be great if you could see the test results of all the smoke test jobs in one screen without having to dig the correct workflow URL out of the error messages?

Now you can: https://app.circleci.com/pipelines/github/adobe/helix-publish/2337/workflows/9a505ea4-ea4f-4625-9ea6-85047d4cca97/jobs/3720/tests

Screen Shot 2020-12-01 at 19 12 06

Sorry, I don't have a test failure to show.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ commands:
             description: the CircleCI token to execute a CircleCI job on the smoke test repo
         remote_branch:
             type: string
-            default: main
+            default: master
```
Comment (trieloff) [inline .circleci/config.yml:40]: @tripodsan the helix pages default is still `master`

Comment (tripodsan) [reply .circleci/config.yml:40]: right. sorry.

File: .circleci/config.yml [hunk 2]
```diff
@@ -98,12 +99,16 @@ commands:
 
             echo "Test Domain is" $TEST_DOMAIN
 
+            npx @adobe/cci-test-aggregator ${pipeline_url}
```
Comment (tripodsan) [inline .circleci/config.yml:102]: where is this ?

Comment (trieloff) [reply .circleci/config.yml:102]: https://github.com/adobe/cci-test-aggregator

**Comment by kptdobe** [comment]
Really good idea...! 
I assume this will work with the helix-smoke-test orb too https://github.com/adobe/helix-continuous/blob/main/.circleci/orbs/helix-smoke-tests/orb.yml

OMG, we run 3206 tests... https://app.circleci.com/pipelines/github/adobe/helix-publish/2340/workflows/5ef89b95-dd5c-43a6-82ac-693580ee6521/jobs/3729/tests

**Review by kptdobe — APPROVED** [review]

**Comment by trieloff** [comment]
> OMG, we run 3206 tests

every good and bad URL is a test against the blocklist.

**Pull request merged**